web/frontend/react.md

```diff
@@ -0,0 +1,159 @@
+
+View Layer: [React](https://facebook.github.io/react/)
+
+Application State Management: [Redux](http://redux.js.org/) + [React-Redux](https://github.com/reactjs/react-redux) ([Tutorial](https://egghead.io/lessons/javascript-redux-the-single-immutable-state-tree))
+
+Javascript: [ES6](https://leanpub.com/understandinges6/read) with [Babel](http://babeljs.io/) compiler
+
+Web Framework: [Express](http://expressjs.com/)
+
+Linter: [ESLint](https://github.com/eslint/eslint)
+
+React Source Code: [Examples](https://react.rocks/tag/Bootstrap)
+
+Module Bundler: [Webpack](https://webpack.github.io/)
+
+[ESLint Loader](https://github.com/MoOx/eslint-loader)
+
+[Babel Loader](https://github.com/babel/babel-loader)
+
+Style Guides:
+
+- [Example 1](https://github.com/airbnb/javascript)
+- [Example 2](https://github.com/airbnb/javascript/tree/master/react)
+
+[The ReactJS Controller View Pattern](http://blog.andrewray.me/the-reactjs-controller-view-pattern/)
+
+## Tutorials
+
+[React.js Tutorial Pt 1: A Comprehensive Guide to Building Apps with React.js](http://tylermcginnis.com/reactjs-tutorial-a-comprehensive-guide-to-building-apps-with-react/)
+[Tyler McGinnis](https://tylermcginnis.com/reactjs-tutorial-a-comprehensive-guide-to-building-apps-with-react/)
+[Sitepoint](http://www.sitepoint.com/how-to-build-a-todo-app-using-react-redux-and-immutable-js/)
+[The Superhero battler app](http://blog.krawaller.se/posts/a-react-redux-example-app/)
+[Teropa](http://teropa.info/blog/2015/09/10/full-stack-redux-tutorial.html)
+[React Tutorial Using the MERN Stack](https://hashnode.com/post/react-tutorial-using-mern-stack-ciiyus9m700qqge53mer0isxz)
+[Full Stack Redux Tutorial](http://teropa.info/blog/2015/09/10/full-stack-redux-tutorial.html)
+[React & Flux](http://survivejs.com/webpack_react/react_and_flux/)
+
+Resources for above tutorials:
+
+- JSX
+  - [In-depth](https://facebook.github.io/react/docs/jsx-in-depth.html)
+  - [Gotchas](https://facebook.github.io/react/docs/jsx-gotchas.html)
+  - [Official site](https://jsx.github.io/)
+
+- Virtual DOM
+  - [Disambiguation](http://jbi.sh/what-is-virtual-dom/)
+  - [Glossary](https://facebook.github.io/react/docs/glossary.html)
+
+- React Router
+  - [Github repo](https://github.com/reactjs/react-router)
+  - [A Comprehensive Introduction](https://www.themarketingtechnologist.co/react-router-an-introduction/)
+
+Key Javascript Concepts
+
+[Scope & Context](http://ryanmorr.com/understanding-scope-and-context-in-javascript/)
+
+[ES6 Fat Arrow Functions](https://strongloop.com/strongblog/an-introduction-to-javascript-es6-arrow-functions/)
+
+[ES6 Spread Operators](http://www.datchley.name/es6-rest-spread-defaults-and-destructuring/)
+
+[ES6 Object Literal Shorthand](http://www.benmvp.com/learning-es6-enhanced-object-literals/)
+
+Other good articles
+
+[Redux Shill](https://blog.andyet.com/2015/08/06/what-the-flux-lets-redux/)
+[MVC context](http://blog.andrewray.me/the-reactjs-controller-view-pattern/)
+
+## Testing
```

Comment:
Are any of our projects not using create-react-app? It makes a lot of these decisions for us and renders some of this mute.